src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseLongTermRetentionBackup.md

```diff
@@ -0,0 +1,332 @@
+
+### BackupName
+```
+Get-AzureRmSqlDatabaseLongTermRetentionBackup [-LocationName] <String> [-ServerName] <String>
+ [-DatabaseName] <String> [-BackupName] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### GetBackupByResourceId
+```
+Get-AzureRmSqlDatabaseLongTermRetentionBackup [-LocationName] <String> [-BackupName] <String>
+ [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GetBackupsByResourceId
+```
+Get-AzureRmSqlDatabaseLongTermRetentionBackup [-LocationName] <String> [-OnlyLatestPerDatabase]
+ [[-DatabaseState] <String>] [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### GetBackupByInputObject
+```
+Get-AzureRmSqlDatabaseLongTermRetentionBackup [-BackupName] <String> [-InputObject] <AzureSqlDatabaseModel>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GetBackupsByInputObject
+```
+Get-AzureRmSqlDatabaseLongTermRetentionBackup [-OnlyLatestPerDatabase] [[-DatabaseState] <String>]
+ [-InputObject] <AzureSqlDatabaseModel> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmSqlDatabaseLongTermRetentionBackup** cmdlet gets all long term retention backups for a location, server, or database or gets a specific long term retention backup.
+
+## EXAMPLES
+
+### Example 1: Get all backups for a location
+```powershell
+PS C:\> Get-AzureRmSqlDatabaseLongTermRetentionBackup -LocationName northeurope
+
+
+BackupExpirationTime : 3/22/2018 11:43:18 PM
+BackupName           : 55970792-164c-4a4a-88e5-7158d092d503;131656309980000000
+BackupTime           : 3/15/2018 11:43:18 PM
+DatabaseName         : database02
+DatabaseDeletionTime : 3/18/2018 4:36:00 PM
+LocationName         : northeurope
+ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server02/longTermRetentionDatabases/database02/longTermRetentionBackups/55970792-164c-4a4a-88e5-7158d092d503;131656309980000000
+ServerName           : server02
+ServerCreateTime     : 2/28/2018 12:12:19 AM
+
+BackupExpirationTime : 3/22/2018 5:50:55 AM
+BackupName           : 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+BackupTime           : 3/15/2018 5:50:55 AM
+DatabaseName         : database01
+DatabaseDeletionTime :
+LocationName         : northeurope
+ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server01/longTermRetentionDatabases/database01/longTermRetentionBackups/601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+ServerName           : server01
+ServerCreateTime     : 2/29/2018 12:12:19 AM
+```
+
+This command gets all long term retention backups for the northeurope location.
```

Comment:
Consider something like "for all servers (which may be live or deleted) in north europe"

Reply:
Changed it to "for all databases (which may be alive or deleted) in north europe" since that's not correct for servers.